tvm/src/schedule/schedule_dataflow_rewrite.cc

```diff
@@ -559,9 +477,12 @@ Array<Tensor> Schedule::move_to(const Tensor& target, Stage parent,
     }
   }
 
-  if (parent.defined()) {  // stream modified tensor
+  // If the parent stage is not empty, it indicates that
+  // a updated tensor is being moved to another scope.
+  // The consumers are directly connected to the parent stages
+  // in this case and we need to re-create consumer stages.
```

Comment:
Not sure if I understand the comment. Any example?

Reply:
This is for hcl.update. I updated the comments.